ua_policy_proposal.md

```diff
@@ -32,11 +31,11 @@ We recommend that browsers supporting First-Party Sets work together to:
 
 # Responsibilities of the Site Author
 
-+   Maintain accuracy in self declaration of common ownership and controllership of the domains listed in a First-Party Set formation request. 
-    +   This means that changes in ownership/controllership must be followed up with a request for changes in the site's First-Party Set within _XX [to be determined]_ days.
-+   Make domain affiliations easily discoverable to the user. As a best practice, site authors should strive to make domain affiliations easily observable to the user, such as through common branding.
++   Maintain accuracy in self declaration of common controllership of user data collected as a result of user interactions with the domains listed in a First-Party Set formation request. 
+    +   This means that changes in controllership must be followed up with a request for changes in the site's First-Party Set within _XX [to be determined]_ days.
++   Make domain affiliations easily discoverable to the user. As a best practice, site authors should strive to make domain affiliations easily observable to the user, such as through common branding. This common branding should be clear to users of assistive technologies.
```

Comment:
Can we remove the mention of assistive technologies here, but keep it in your second reference (line 76)? The sentences fits in better with the specificity of guidance around common group identity.

Reply:
Done (in 3c3e485)